neutron/neutron.go

```diff
@@ -251,12 +253,14 @@ func (c *Client) DeleteFloatingIPV2(ipId string) error {
 
 // PortV2 describes a defined network for administrating a port.
 type PortV2 struct {
+	AdminStateUp        bool             `json:"admin_state_up,omitempty"`
 	Description         string           `json:"description,omitempty"`
 	DeviceId            string           `json:"device_id,omitempty"`
+	DeviceOwner         string           `json:"device_owner,omitempty"`
 	FixedIPs            []PortFixedIPsV2 `json:"fixed_ips,omitempty"`
 	Id                  string           `json:"id,omitempty"`
 	Name                string           `json:"name,omitempty"`
-	NetworkId           string           `json:"network_id"`
+	NetworkId           string           `json:"network_id,omitempty"`
```

Comment:
If you try and update the Port with an empty `NetworkId` it will fail with an error. Instead, we have to omit empty even for other methods. V3 of the library should look into Port types for method verb requests.